src/modules/ffmpeg/producer/av_input.cpp

```diff
@@ -126,7 +126,7 @@ void Input::internal_reset()
 
     static const std::set<std::wstring> PROTOCOLS_TREATED_AS_FORMATS = {L"dshow", L"v4l2", L"iec61883"};
 
-    AVInputFormat* input_format = nullptr;
+    const AVInputFormat* input_format = nullptr;
```

Comment:
Error	C2664	'int avformat_open_input(AVFormatContext **,const char *,AVInputFormat *,AVDictionary **)': cannot convert argument 3 from 'const AVInputFormat *' to 'AVInputFormat *'	ffmpeg	C:\Code\casparcg\official-master\src\modules\ffmpeg\producer\av_input.cpp	158